tests/product/test_collect.py

```diff
@@ -67,13 +67,6 @@ def log_msg(self, actual, expected):
         msg = '%s != %s' % (actual, expected)
         return msg
 
-    @attr('smoketest')
```

Comment:
This can be removed because `test_system_info_pa_separate_node()` tests a very similar scenario.